library/bignum_mod_raw.h

```diff
@@ -233,6 +232,24 @@ int mbedtls_mpi_mod_raw_to_mont_rep( mbedtls_mpi_uint *X,
  */
 int mbedtls_mpi_mod_raw_from_mont_rep( mbedtls_mpi_uint *X,
                                        const mbedtls_mpi_mod_modulus *m );
+
+/** \brief  Perform fixed width modular negation.
+ *
+ * The size of the operation is determined by \p N. \p A must have
+ * the same number of limbs as \p N.
+ *
+ * \p X may be aliased to \p A. \p A and \p X must have
+ * the same number of limbs as the modulus
```

Comment:
```suggestion
 * The size of the operation is determined by \p N: both \X and \A
 * must have the same number of limbs as \p N.
 *
 * \p X may be aliased to \p A.
```

Reply:
Ammended the documentation commit to include that suggestion